crates/fluvio-cluster/src/runtime/local/etcd.rs

```diff
@@ -0,0 +1,47 @@
+use std::{
+    fs::File,
+    path::PathBuf,
+    process::{Command, Stdio},
+};
+
+use fluvio_command::CommandExt;
+use tracing::info;
+
+use super::LocalRuntimeError;
+
+pub struct EtcdProcess {
```

Comment:
`pub(crate)`.  I know we were not consistent in this

Reply:
Done